server/src/modules/users/infra/repos/auth-secret-repo/implementations/mikro-auth-secret-repo.ts

```diff
@@ -9,9 +9,30 @@ import { AuthSecretRepo } from '../../auth-secret-repo/auth-secret-repo'
 export class MikroAuthSecretRepo implements AuthSecretRepo {
   constructor(protected authSecretEntityRepo: EntityRepository<AuthSecretEntity>) {}
 
+  async exists(clientId: string): Promise<Result<boolean, DBErrors>> {
+    const authSecretEntity = await this.authSecretEntityRepo.findOne({ 
+      clientId: clientId, 
+    })
+    if(authSecretEntity !== null){
+      return Result.ok(authSecretEntity !== null)
+    } else {
+      return Result.err(new DBError.AuthSecretNotFoundError(clientId))
+    }
+  }
```

Comment:
I'm pretty sure this function isn't doing what's intended. If the repo can't find an authSecretEntity, shouldn't it return `false` instead of an error? Same comment also applies to `Mock` repo.

Reply:
Yep, good call.